src/handles/handleDiagnostic.ts

```diff
@@ -267,12 +271,12 @@ async function handleLinter (
         range: {
           start: {
             // line and character is base zero so need -1
-            line: sumNum(line, -1, offsetLine),
-            character: sumNum(column, -1, offsetColumn)
+            line: sumNum(line, -1, offsetLine + offsetLineStart),
+            character: sumNum(column, -1, offsetColumn + offsetColumnStart)
           },
```

Comment:
Why this is `offsetLine + offsetLineStart` and `offsetColumn + offsetColumnStart`